docs/r/cdap_application.md

```diff
@@ -4,14 +4,13 @@
 
 # Example
 
+Note, `jsonencode` is used for the purpose of explicit example.
+In most real use cases you would just use.
+`spec = file("${path.module}/realtive/path/to/pipeline_spec.json")`
 ```
 resource "cdap_application" "pipeline" {
     name = "example_pipeline"
-    artifact {
-        name = "cdap-data-pipeline"
-        version = "6.1.1"
-    }
-    config = jsonencode({
+    spec = jsonencode({
```

Comment:
p.s. this example should ideally work out of the box and it won't anymore. I can fix this later but if you have a chance we need to shift the name and artifact into the jsonencode call and move the existing fields under the "config" block

Reply:
fixed.